middlewares/validateJWT.js

```diff
@@ -0,0 +1,30 @@
+const moment = require('moment');
+const jwt = require("jsonwebtoken");
+
+const ensureAuthenticated = (req, res, next) => {
+    if(!req.headers.authorization) {
+        return res.status(403).json({
```

Comment:
Cambiemos por el sugerido en la tarea

> Status: 401 Unauthorized
Content-Type: application/json
ResponseBody: {
  "message": "Not authorized"
}